pallets/popi/src/lib.rs

```diff
@@ -113,6 +136,15 @@ pub mod pallet {
 		GraphicDesign,
 	}
 
+	#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, Debug, Clone, Copy)]
+	#[scale_info(skip_type_params(T))]
+	pub struct UniqueProjectIdentifier<T: Config> {
+		approver: T::AccountId,
+		worker: T::AccountId,
+		board_id: u32,
```

Comment:
I think we should create a generic type for `board_id` and `task_id`, instead of using u32. Obviously it won't make a difference right now, but using the generic will allow us to easily make an adjustment in the future, when we decide how we want to tackle this.

Something like this:
```suggestion
		board_id: BoardId,
```

If you disagree, just keep it as is. It's just my personal opinion.

Reply:
I definitively agree.

Reply:
now I remember why I didn't do it, it is an abstract concept. I'm waiting for the boards storage to be implemented so that we can refer to it.